doc/introduction/measurements.rst

```diff
@@ -129,9 +130,11 @@ and :func:`~.pennylane.sample`.
 Counts
 ------
 
-To avoid dealing with long arrays for the larger numbers of shots, one can pass an argument counts=True
-to :func:`~pennylane.sample`. In this case, the result will be a dictionary containing the number of occurrences for each
-unique sample. The previous example will be modified as follows:
+To avoid dealing with long arrays for the larger numbers of shots, one can use :func:`~pennylane.counts` rather than
+ :func:`~pennylane.sample`. This performs the same measurement as sampling, but returns a dictionary containing the 
+ possible measurement outcomes and the number of occurrences for each, rather than a list of all outcomes. 
+
+ The previous example will be modified as follows:
```

Comment:
```suggestion
:func:`~pennylane.sample`. This performs the same measurement as sampling, but returns a dictionary containing the 
possible measurement outcomes and the number of occurrences for each, rather than a list of all outcomes. 

The previous example will be modified as follows:
```